packages/checkout/widgets-lib/src/components/Transactions/NotEnoughEthToWithdraw.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Body,
+  Box, Button, CloudImage, Drawer, Heading,
+} from '@biom3/react';
+import { CHECKOUT_CDN_BASE_URL } from 'lib';
+import { Environment } from '@imtbl/config';
+import { useTranslation } from 'react-i18next';
+import {
+  actionButtonContainerStyles, actionButtonStyles, containerStyles, contentTextStyles,
+} from './NotEnoughEthToWithdrawStyles';
+
+export interface NotEnoughEthToWithdrawProps {
+  visible: boolean,
+  onClose: () => void,
+  onChangeAccount: () => void,
+}
+
+export function NotEnoughEthToWithdraw({
+  visible,
+  onClose,
+  onChangeAccount,
+}: NotEnoughEthToWithdrawProps) {
+  const { t } = useTranslation();
+  const ethLogo = `${CHECKOUT_CDN_BASE_URL[Environment.PRODUCTION]}/v1/blob/img/tokens/eth.svg`;
```

Comment:
using `PRODUCTION` env here - can we get the env instead from the checkout object on context

Reply:
Yeah to be honest I thought this was more work than what it was worth. I doubt that we are going to change image here based on environment, that's why I left it out.

Reply:
But then again, we may decide to update images in the future and we may want to test them in sandbox first? what do you think?

Reply:
i think testing symbols in sandbox is a good reason to use env from checkout obj